tests/osaka/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpi.py

```diff
@@ -1002,3 +1644,183 @@ def test_double_rjumpi_invalid_max_stack_height(
         ),
         expect_exception=EOFException.INVALID_MAX_STACK_HEIGHT,
     )
+
+
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="backwards_rjumpi_10",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH1[190]
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[1]
+                    + Op.POP
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-11]
+                    + Op.STOP,
+                    max_stack_height=3,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001000e040000000080000360be6000e10001506000e1fff500",
+        ),
+        Container(
+            name="backwards_rjumpi_3",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.POP
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-7]
+                    + Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-13]
+                    + Op.STOP,
+                    max_stack_height=2,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001000e04000000008000025f506000e1fff95f6000e1fff300",
+        ),
+        Container(
+            name="backwards_rjumpi_5",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[1]
+                    + Op.ADD
+                    + Op.DUP1
+                    + Op.DUP1
+                    + Op.RJUMPI[-8]
+                    + Op.STOP,
+                    max_stack_height=2,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001000a04000000008000025f6001018080e1fff800",
+        ),
+        Container(
+            name="backwards_rjumpi_8",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.POP
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-7]
+                    + Op.PUSH0
+                    + Op.RJUMP[-11],
```

Comment:
Minor nitpick: The issue here is not really the `RJUMPI` rather the `RJUMP`?

Not saying that we should remove it from here, but just wanted to point it out in case it was not the purpose of the test.

Reply:
I think the test should be `RJUMPI[-6]` (invalid) followed by `RJUMP[-10]` (valid). So I will correct it here, and the original one I will move to `test_rjump.py`.